ml/src/test/scala/ai/h2o/sparkling/ml/algos/MultinomialPredictionTestSuite.scala

```diff
@@ -113,4 +113,15 @@ class MultinomialPredictionTestSuite extends FunSuite with Matchers with SharedH
     assert(schema == expectedSchema)
     assert(schema == expectedSchemaByTransform)
   }
+
+  test("getDomainValues") {
```

Comment:
In fact, this test is testing `getDomainValuesForCol` method. Shouldn't we change name of this test and introduce another one for  `getDomainValues`?

Reply:
👍  Good catch, I should introduce more tests anyway